concepticondata/references/references.bib

```diff
@@ -3305,6 +3305,27 @@ @Article{Dellert2018
   Year                                = {2018}
 }
 
+@incollection{Pharao2020,
+	title={¿Familia o vecinos? Investigando la relación entre el proto-náhuatl y el proto-corachol [Family or neighbors? Investigating the relation between Proto-Náhuatl and Proto-Corachol]},
+	author = {Pharao Hansen, Magnus},
+	year={2020},
+	booktitle={Lenguas yutoaztecas: historia, estructuras y contacto lingüístico. Homenaje a Karen Dakin},
+	editor={Yañez Rosales, Rosa H.},
+	publisher={Universidad de Guadalajara},
+	address = {Guadelajara},
```

Comment:
Should the address be Guadalajara as well?

Reply:
Yes, it's a city where the University of Guadalajara is located.

Reply:
I know it's a city ;) Sorry, did not make myself clear enough. The university is called GuadAlajara while the address is currently GuadElajara. I think that's a typo (or are there genuinely two ways of writing the name?)

Reply:
You're absolutely right!